services/dis-console-ui/src/styles.css

```
@@ -0,0 +1,931 @@
+body {
+  margin: 0;
+  background: var(--ds-color-neutral-background-default, #fff);
+  font-family: var(--ds-font-family, 'Inter'), 'Inter', system-ui, sans-serif;
```

Stylelint (17.14.1)

[error] Expected no quotes around "Inter" (font-family-name-quotes)

(font-family-name-quotes)